Packs/PrismaSaasSecurity/ModelingRules/SaasSecurityEventCollector.yml

```diff
@@ -0,0 +1,4 @@
+name: SaasSecurityEventCollector
+id: SaasSecurityEventCollector
+fromversion: 3.3.0
+rules: ''
```

Comment:
You are missing a schema see this https://panw-global.slack.com/archives/C03DED0TARL/p1653312570665359

Reply:
done